web/chromecom.js

```diff
@@ -338,8 +339,9 @@ class ChromePreferences extends BasePreferences {
 
 var ChromeExternalServices = Object.create(DefaultExternalServices);
 ChromeExternalServices.initPassiveLoading = function (callbacks) {
-  var appConfig = PDFViewerApplication.appConfig;
-  ChromeCom.resolvePDFFile(appConfig.defaultUrl,
+  let appConfig = PDFViewerApplication.appConfig;
+  let overlayManager = PDFViewerApplication.overlayManager;
```

Comment:
This can be `let { appConfig, overlayManager, } = PDFViewerApplication;` instead.